express-gateway/README-short.txt

```diff
@@ -0,0 +1 @@
+The official release of Express-Gateway.
```

Comment:
This is the text that will show up when folks do `docker search express-gateway` and on https://hub.docker.com/search?q=express-gateway&type=image -- this description is fine, but it doesn't really tell folks anything _about_ express gateway (which is OK by us, but you might want something better so folks know what it is and might try it out).